src/Quartz/Impl/AdoJobStore/StdAdoDelegate.Calendars.cs

```diff
@@ -1,4 +1,5 @@
 using Microsoft.Extensions.Logging;
+using System.Data;
```

Comment:
the integration tests seem to be showing a real problem, now different extension method is being chosen for `reader.GetString`, I think you need to remove `using System.Data;` additions in this PR and instead refer to `SequentialAccess` with its full name `System.Data.CommandBehavior.SequentialAccess`

Reply:
I couldn't easily push a tweak to to this PR as you have created your PR for main - it's better convention to have PR's made from separate branch

Reply:
Ok, that makes a lot more sense. I am removing the usings that I added now and rolling back the test